qiskit_nature/drivers/qmolecule.py

```diff
@@ -49,15 +49,13 @@ class QMolecule:
 
     def __init__(self, filename=None):
         warn_deprecated(
-            "0.2.0",
-            DeprecatedType.CLASS,
-            "QMolecule",
-            additional_msg=(
-                "Instead look towards the qiskit_nature.properties.second_quantization.electronic "
-                "module. The new return object for drivers is the ElectronicStructureDriverResult "
-                "which you can construct from a QMolecule via the `from_legacy_driver_result()` "
-                "method."
-            ),
+            "0.5.0",
+            old_type=DeprecatedType.CLASS,
+            old_name="qiskit_nature.drivers.QMolecule",
+            new_type=DeprecatedType.CLASS,
+            new_name="qiskit_nature.second_q.QMolecule",
```

Comment:
This class will also be removed as part of #748. No need to update the deprecation here